lib/assets/models/index.js

```diff
@@ -5,7 +5,7 @@ var path      = require('path');
 var Sequelize = require('sequelize');
 var basename  = path.basename(module.filename);
 var env       = process.env.NODE_ENV || 'development';
-var config    = require(<%= configFile %>)[env];
```

Comment:
This change will break https://github.com/sequelize/cli/blob/22f208e7fda83268a8ae003b1074a12573befd83/lib/helpers/init-helper.js#L72